include/mbedtls/md.h

```diff
@@ -233,8 +234,8 @@ mbedtls_md_type_t mbedtls_md_get_type( const mbedtls_md_info_t *md_info );
 const char *mbedtls_md_get_name( const mbedtls_md_info_t *md_info );
 
 /**
- * \brief           This function Extracts the message-digest name from the  
- *                  message-digest information structure. 
+ * \brief           This function calculates the  media-digest checksum 
```

Comment:
Blocker: Typo, and the documentation doesn't refer to `md_starts`. 
Suggestion: Make it uniform with the other SHA-XXX files, e.g. `This function starts a message-digest computation.`

Reply:
Fixed